stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -326,12 +341,15 @@ public final class _ExecutionContext {
       debugLog(
         "Device \(deviceId) has type \(deviceType) and name \(deviceName)."
       )
+			if deviceType == "CPU" && deviceName == localCPUDeviceName {
```

Comment:
In this code base we slightly prefer comma when an `if` has multiple conditions.

```suggestion
			if deviceType == "CPU", deviceName == localCPUDeviceName {
```

Reply:
Done.